src/dump_atom.cpp

```diff
@@ -166,11 +166,11 @@ int DumpAtom::modify_param(int narg, char **arg)
   }
 
   if (strcmp(arg[0],"triclinic/general") == 0) {
-    triclinic_general = 1;
+    if (narg < 2) error->all(FLERR,"Illegal dump_modify command");
+    triclinic_general = utils::logical(FLERR,arg[1],false,lmp);
     if (triclinic_general && !domain->triclinic_general)
-      error->all(FLERR,"Dump_modify triclinic/general cannot be used "
-                 "if simulation box is not general triclinic");
-    return 1;
+      error->all(FLERR,"Dump_modify triclinic/general invalid b/c simulation box is not");
```

Comment:
Can you use the old error message from dump_atom.cpp here and also in dump_custom.cpp ?  I think it is more clear than this one.

Reply:
I Updated both messages. The old one is definitely more useful to the user.